src/main/java/mel/main/Mel.java

```diff
@@ -0,0 +1,70 @@
+package mel.main;
+
+import mel.exceptions.MelException;
+import mel.exceptions.TaskException;
+import mel.tasks.TaskList;
+import mel.utils.Storage;
+import mel.utils.UI;
+
+import java.util.Scanner;
+
+/**
+ * Mel class is the main class of Mel chatbot.
+ */
+public class Mel {
+    private final TaskList taskList;
+    private final UI ui;
+
+    /**
+     * Constructor for Mel chatbot, instantiates
+     * TaskList, Storage and UI elements.
+     */
+    public Mel() {
+        Storage storage = new Storage();
+        taskList = new TaskList(this, storage);
+        ui = new UI(this);
```

Comment:
UI should be Ui instead, follow the naming convention [here](https://se-education.org/guides/conventions/java/intermediate.html#naming).